src/components/search-textbox.vue

```diff
@@ -0,0 +1,58 @@
+<!--
+Copyright 2025 ODK Central Developers
+See the NOTICE file at the top-level directory of this distribution and at
+https://github.com/getodk/central-frontend/blob/master/NOTICE.
+
+This file is part of ODK Central. It is subject to the license terms in
+the LICENSE file found in the top-level directory of this distribution and at
+https://www.apache.org/licenses/LICENSE-2.0. No part of ODK Central,
+including this file, may be copied, modified, propagated, or distributed
+except according to the terms contained in the LICENSE file.
+-->
+
+<template>
+  <label class="form-group">
+    <input v-model="searchTextbox" class="form-control search-textbox" :placeholder="label"
+      :aria-label="label" :aria-disabled="disabled" autocomplete="off" @keydown.enter="setSearchTerm" @focusout="revert">
+    <button v-show="searchTextbox" type="button" class="close"
+      :aria-label="$t('action.clearSearch')" @click="clearSearch">
```

Comment:
I'm realizing that `action.clearSearch` doesn't exist in en.json5. It's in `ProjectUserList`. How about removing it from `ProjectUserList` and adding it to this component? You can use `@transifexKey` to prevent it from changing in Transifex and being retranslated.